### PR TITLE
feat: Refactor Order form UI and allow setting custom expiry time

### LINKED ISSUE #1492: Add support for custom order expiry time in the order-form
**Is your feature request related to a problem? Please describe.**

Now that the event organizer can decide on what information to collect from each user, we should also allow them to set custom order expiry time.

**Additional context**

Parent #1452

### PR DESCRIPTION
**Pull request by srv-twry**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Now that the event organizer can decide on what information to collect from each user, we should also allow them to set custom order expiry time.

#### Changes proposed in this pull request:
Refactored the UI for the order form and allowed setting custom order expiry time.

Fixes #1492

### CODE REVIEW

File: app/templates/events/view/tickets/order-form.hbs [hunk 1]
```diff
@@ -1,66 +1,83 @@
...
+  <h3 class="ui dividing header">
+    <i class="options box icon"></i>
+    <div class="content">
+      {{t 'Registration Options'}}
+    </div>
+  </h3>
+  <div class="field">
+    <label>{{t 'REGISTRATION TIME LIMIT'}}</label>
+    <div class="{{unless device.isMobile 'two wide'}} field">
+      {{input type='number' name='orderExpiryTime' value=model.event.orderExpiryTime}}
```
Comment (pradeepgangwar) [inline app/templates/events/view/tickets/order-form.hbs]: Should we have a limit on max value here?

Comment (srv-twry) [reply app/templates/events/view/tickets/order-form.hbs]: Yeah alright, I'll add it in the next PR. I also have to limit the input to only allow positive numbers. Need to use JS for that. Will also take care of this.

**Comment by srv-twry** [comment]
@pradeepgangwar please merge.

**Comment by pradeepgangwar** [comment]
@srv-twry Create a follow up issue.

**Pull request merged**